src/bitbucket/model.ts

```diff
@@ -221,7 +221,7 @@ export type CreatePullRequestData = {
     closeSourceBranch: boolean;
 };
 
-export type ApprovalStatus = 'APPROVED' | 'UNAPPROVED' | 'NEEDS_WORK';
+export type ApprovalStatus = 'APPROVED' | 'UNAPPROVED' | 'CHANGES_REQUESTED' | 'REQUEST_CHANGES';
```

Comment:
`REQUEST_CHANGES` Doesn't seem like an approval status but an action. Could you explain the difference here?